app/src/main/java/mozilla/lockbox/presenter/WelcomePresenter.kt

```diff
@@ -19,15 +21,28 @@ interface WelcomeView {
 
 class WelcomePresenter(
     private val view: WelcomeView,
-    private val dispatcher: Dispatcher = Dispatcher.shared
+    private val dispatcher: Dispatcher = Dispatcher.shared,
+    private val fingerprintStore: FingerprintStore = FingerprintStore.shared
 ) : Presenter() {
     override fun onViewReady() {
+        val isDeviceSecure: Boolean = fingerprintStore.isKeyguardDeviceSecure
```

Comment:
why are you introducing a boolean to the presenter? could just be inline on ln 33.